Rubberduck.Core/UI/Command/MenuItems/CommandBars/SerializeProjectsCommandMenuItem.cs

```diff
@@ -76,6 +76,16 @@ protected override void OnExecute(object parameter)
                 Logger.Warn($"Serializing {library.Path}.");
                 _serializationProvider.SerializeProject(library);
             }
+
+#if DEBUG
+            //Dumb hack cos I'm too lazy to wire up another button...
+            var traceDirectory = Path.Combine(Path.GetDirectoryName(_serializationProvider.Target), "COM Trace");
```

Comment:
Nitpick. Can this go in its own folder? I don't want to forget about it if we eventually decide to perform some build actions around the serialized `_serializationProvider.Target`.

Reply:
It is already going into its own folder called `COM Trace`? Besides, even if we went with build action, we would not need to do this via the `CommandMenuItem` but rather the `Serialize***` method for the respective classes, no?

Reply:
Own folder as in not a sub-folder of the serialized projects path (which should probably be named something other than Declarations now, but that's on me...).  Like I said, nitpick.